### PR TITLE
Add #exp_points to CourseUser model

### PR DESCRIPTION
**Pull request by kxmbrian**
Merge #173 first.

### CODE REVIEW

File: spec/models/course_user_spec.rb [hunk 1]
```diff
@@ -11,4 +12,17 @@
     it { is_expected.to be_student }
     it { is_expected.not_to be_phantom }
   end
+
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    describe '.exp_points' do
```
Comment (allenwq) [inline spec/models/course_user_spec.rb]: <code>#</code> for instance methods  and <code>.</code> for class methods
http://betterspecs.org/#describe

File: spec/models/course_user_spec.rb [hunk 2]
```diff
@@ -11,4 +12,17 @@
...
+      before do
+        exp_record_1 = FactoryGirl.create :manual_exp_record
```
Comment (allenwq) [inline spec/models/course_user_spec.rb]: We have required <code>FactoryGirl</code>  so <code>create</code> is enough

File: spec/models/course_user_spec.rb [hunk 3]
```diff
@@ -11,4 +12,17 @@
...
+        @course_user = exp_record_1.course_user
+        exp_record_2 = FactoryGirl.create :manual_exp_record, course_user: @course_user
+        @exp_total = exp_record_1.exp_awarded + exp_record_2.exp_awarded
+      end
+      it { expect(@course_user.exp_points).to eq @exp_total }
```
Comment (allenwq) [inline spec/models/course_user_spec.rb]: Use subject here

http://betterspecs.org/#subject

**Comment by kxmbrian** [comment]
Replaced by #186, which is PR-ed from a name-spaced branch instead. Also, a cherry-picked commit was added.

**Pull request closed without merging**